src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate parsedDateFrom = LocalDate.parse(dateFrom, formatter);
+        LocalDate parsedDateTo = LocalDate.parse(dateTo, formatter);
+        StringBuilder stringBuilder = new StringBuilder();
+
+        stringBuilder.append("Report for period ")
+                .append(dateFrom).append(" - ")
+                .append(dateTo)
+                .append(System.lineSeparator());
```

Comment:
one method should have one purpose - you can make separate method with this fuctionality